src/common/QLogging.cpp

```diff
@@ -10,8 +10,8 @@ Q_LOGGING_CATEGORY(chatterinoApp, "chatterino.app", logThreshold);
 Q_LOGGING_CATEGORY(chatterinoArgs, "chatterino.args", logThreshold);
 Q_LOGGING_CATEGORY(chatterinoBenchmark, "chatterino.benchmark", logThreshold);
 Q_LOGGING_CATEGORY(chatterinoBttv, "chatterino.bttv", logThreshold);
-Q_LOGGING_CATEGORY(chatterinoCommon, "chatterino.cache", logThreshold);
-Q_LOGGING_CATEGORY(chatterinoCache, "chatterino.common", logThreshold);
+Q_LOGGING_CATEGORY(chatterinoCommon, "chatterino.common", logThreshold);
+Q_LOGGING_CATEGORY(chatterinoCache, "chatterino.cache", logThreshold);
```

Comment:
The entries seem to be sorted alphabetically.
```suggestion
Q_LOGGING_CATEGORY(chatterinoCache, "chatterino.cache", logThreshold);
Q_LOGGING_CATEGORY(chatterinoCommon, "chatterino.common", logThreshold);
```